### PR TITLE
latest Zanata translation changes in kie-wb-distributions from 2016-11-30

### PR DESCRIPTION
**Pull request by kiereleaseuser**

### CODE REVIEW

File: kie-drools-wb/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/client/resources/i18n/LoginConstants_fr.properties [hunk 1]
```diff
@@ -1,18 +1,10 @@
-# translation auto-copied from project KIE webapp distributions, version 6.0.0, document org.kie/kie-drools-wb-webapp/org/kie/workbench/drools/client/resources/i18n/LoginConstants, author nmirasch
-UserName=Nom d''utilisateur
-# translation auto-copied from project KIE webapp distributions, version 6.0.0, document org.kie/kie-drools-wb-webapp/org/kie/workbench/drools/client/resources/i18n/LoginConstants, author nmirasch
+UserName=Nom d'utilisateur
```
Comment (manstis) [inline kie-drools-wb/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/client/resources/i18n/LoginConstants_fr.properties]: @mbiarnes Did the replacer plugin run?

File: kie-drools-wb/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/client/resources/i18n/LoginConstants_fr.properties [hunk 2]
```diff
@@ -1,18 +1,10 @@
...
-loginAsAnotherUser=Connectez-vous en tant qu''un autre utilisateur
-# translation auto-copied from project KIE webapp distributions, version 6.2.0, document org.kie/kie-drools-wb-webapp/org/kie/workbench/drools/client/resources/i18n/LoginConstants, author croe@redhat.com
+loginFailed=La connexion a \u00e9chou\u00e9 \: non autoris\u00e9
+loginAsAnotherUser=Connectez-vous en tant qu'autre utilisateur
 logoutSuccssful=D\u00e9connexion r\u00e9ussie
-# translation auto-copied from project KIE webapp distributions, version 6.2.0, document org.kie/kie-drools-wb-webapp/org/kie/workbench/drools/client/resources/i18n/LoginConstants, author croe@redhat.com
 loginAgain=Connectez-vous \u00e0 nouveau
 loadingPleaseWait=Veuillez patienter
-loadingApplication=Chargement de l''application...
+loadingApplication=Chargement de l'application...
```
Comment (manstis) [inline kie-drools-wb/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/client/resources/i18n/LoginConstants_fr.properties]: Did the replacer plugin run?

File: kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties [hunk 3]
```diff
@@ -1,18 +1,10 @@
-# translation auto-copied from project KIE webapp distributions, version 6.0.0, document org.kie/kie-wb-webapp/org/kie/workbench/client/resources/i18n/LoginConstants, author nmirasch
-UserName=Nom d''utilisateur
-# translation auto-copied from project KIE webapp distributions, version 6.0.0, document org.kie/kie-wb-webapp/org/kie/workbench/client/resources/i18n/LoginConstants, author nmirasch
+UserName=Nom d'utilisateur
```
Comment (manstis) [inline kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties]: Did the replacer plugin run?

File: kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties [hunk 4]
```diff
@@ -1,18 +1,10 @@
...
-loginAsAnotherUser=Connectez-vous en tant qu''un autre utilisateur
-# translation auto-copied from project KIE webapp distributions, version 6.2.0, document org.kie/kie-wb-webapp/org/kie/workbench/client/resources/i18n/LoginConstants, author croe@redhat.com
+loginFailed=La connexion a \u00e9chou\u00e9 \: non autoris\u00e9
+loginAsAnotherUser=Connectez-vous en tant qu'autre utilisateur
 logoutSuccssful=D\u00e9connexion r\u00e9ussie
-# translation auto-copied from project KIE webapp distributions, version 6.2.0, document org.kie/kie-wb-webapp/org/kie/workbench/client/resources/i18n/LoginConstants, author croe@redhat.com
 loginAgain=Connectez-vous \u00e0 nouveau
 loadingPleaseWait=Veuillez patienter
-loadingApplication=Chargement de l''application...
+loadingApplication=Chargement de l'application...
```
Comment (manstis) [inline kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties]: Did the replacer plugin run?

**Comment by mbiarnes** [comment]
@manstis the replacer plugin run - yepp - twice:

after mvn zanata:pull-module and after mvn zanata:pullmodule -Dproductized
mvn replacer:replace -N
And both runs without errors!

File: kie-drools-wb/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/client/resources/i18n/LoginConstants_fr.properties (hunk 1 above)
Comment (mbiarnes) [inline kie-drools-wb/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/client/resources/i18n/LoginConstants_fr.properties]: yes

File: kie-drools-wb/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/client/resources/i18n/LoginConstants_fr.properties (hunk 2 above)
Comment (mbiarnes) [inline kie-drools-wb/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/client/resources/i18n/LoginConstants_fr.properties]: yes

File: kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties (hunk 3 above)
Comment (mbiarnes) [inline kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties]: yes

**Pull request merged**